### DISCUSSION TITLE
My profile is not visible to others

### DISCUSSION THREAD

**Discussion by raj-pragadeesh** [community/community#87308] [New to GitHub]
My GitHub profile https://github.com/raj-pragadeesh is not visible to others. My public projects are also not visible. My PRs are not visible to the users in the same project. If my username is searched, other users cannot find my profile.

**Comment by murat-yasar** [comment]
your profile is visible, indeed (check out: https://github.com/search?q=raj-pragadeesh&type=users). After the search, people must click on the users or discussion icons on the table left. Because you have only an account and a issue. 

I suggest you have a strong profile page. You can check mine: [murat-yasar](https://github.com/murat-yasar)
I can help you to create one.

**Reply by MrMadaraUchiha** [reply]
I have the same problem, could you help me, please, my profile has an error 404 for people.

**Comment by PD1LOI** [comment]
https://github.com/PD1LOI is not visible to everyone. When I click on that link I see a 404 website. Only when I log in do I see my site. But for others it remains invisible.

**Comment by PD1LOI** [comment]
Good morning, How come I can't see Github if I don't log in? Before a while back I could just see the Github site. Now I get "404 Error" website not available. Fellow users who want to see my work receive the same message and that is very difficult to Push repros

**Comment by treshanappuhamy** [comment]
try changing the visibility of the profile to PUBLIC

**Comment by jfullstackdev** [comment]
it's visible by now even if it's in Incognito

![image](https://github.com/community/community/assets/47092464/88916d4a-5ded-4aab-a547-f2ff2201b50c)

**Comment by vuphuong1794** [comment]
https://github.com/vuphuong1794 is not visible to everyone. When I click on that link I see a 404 website. Only when I log in do I see my site. But for others it remains invisible.
